core/point/point_test.go

```diff
@@ -13,3 +13,117 @@ func TestCreate(t *testing.T) {
 		t.Errorf("p.Y()=%v, expected %v", gotY, 2)
 	}
 }
+
+func TestPointSGFTranslation(t *testing.T) {
+	type Point struct {
+		x int64
+		y int64
+	}
+
+	// First test translation from integer-point to SGF-string-point
+	testToSGFCases := []struct {
+		desc string
+		in   Point
+		want string
+	}{
+		{
+			desc: "Point => SGF",
+			in: Point{
+				x: 8,
+				y: 16,
+			},
+			want: "iq",
+		},
+		{
+			desc: "Point => SGF",
+			in: Point{
+				x: 12,
+				y: 5,
+			},
+			want: "mf",
+		},
+		{
+			desc: "Point => SGF",
+			in: Point{
+				x: 33,
+				y: 8,
+			},
+			want: "Hi",
+		},
+		{
+			desc: "Point => SGF",
+			in: Point{
+				x: 40,
+				y: 51,
+			},
+			want: "OZ",
+		},
+	}
+
+	for _, tc := range testToSGFCases {
+		t.Run(tc.desc, func(t01 *testing.T) {
+			toSGFOut := New(tc.in.x, tc.in.y).ToSGF()
+			if t01 == nil || toSGFOut != tc.want {
```

Comment:
why would t01 be nil? that should be impossible -- not something you need to check for. It's ok to let such a nil pointer exception/panic (NPE) happen here.